app/services/host.py

```diff
@@ -249,7 +249,7 @@ def _is_expired(self, host):
         """
 
         last_check_in = host['last_check_in']
-        now = datetime.datetime.now()
+        now = datetime.datetime.utcnow()
```

Comment:
good find.

let's wrap this in pytz.utc.localize and remove lines 256, 257

Reply:
Cool. Amended.